log/logger.go

```diff
@@ -123,26 +123,26 @@ func (f Fields) Len() int { return len(f) }
 func (f Fields) ValueAt(i int) Field { return f[i] }
 
 // NullLogger is a logger that emits nowhere.
-var NullLogger Logger = nullLogger{}
+var NullLogger Logger = &nullLogger{}
 
 type nullLogger struct {
 	fields Fields
 }
 
-func (nullLogger) Enabled(_ Level) bool                   { return false }
-func (nullLogger) Fatalf(msg string, args ...interface{}) {}
-func (nullLogger) Fatal(msg string)                       { os.Exit(1) }
-func (nullLogger) Errorf(msg string, args ...interface{}) {}
-func (nullLogger) Error(msg string)                       {}
-func (nullLogger) Warnf(msg string, args ...interface{})  {}
-func (nullLogger) Warn(msg string)                        {}
-func (nullLogger) Infof(msg string, args ...interface{})  {}
-func (nullLogger) Info(msg string)                        {}
-func (nullLogger) Debugf(msg string, args ...interface{}) {}
-func (nullLogger) Debug(msg string)                       {}
-func (l nullLogger) Fields() LoggerFields                 { return l.fields }
-
-func (l nullLogger) WithFields(newFields ...Field) Logger {
+func (*nullLogger) Enabled(_ Level) bool                   { return false }
+func (*nullLogger) Fatalf(msg string, args ...interface{}) {}
```

Comment:
why not os.Exit(1) here too?

Reply:
Not sure, that wasn't my change, I can add it however.

Reply:
Sounds good